package.json

```diff
@@ -24,22 +25,22 @@
   },
   "license": "MIT",
   "peerDependencies": {
-    "@angular/common": ">6.0.0",
-    "@angular/core": ">6.0.0",
-    "@angular/platform-browser": ">6.0.0",
-    "rxjs": ">6.0.0"
+    "@angular/common": ">10.0.0",
+    "@angular/core": ">10.0.0",
+    "@angular/platform-browser": ">10.0.0",
+    "rxjs": ">6.6.0"
   },
   "devDependencies": {
-    "@angular/common": "latest",
-    "@angular/compiler": "latest",
-    "@angular/compiler-cli": "latest",
-    "@angular/core": "latest",
-    "@angular/platform-browser": "latest",
-    "@angular/platform-server": "latest",
-    "ng-packagr": "^4.7.1",
-    "rxjs": "latest",
-    "tsickle": "^0.34.3",
-    "typescript": "latest",
-    "zone.js": "latest"
+    "@angular/common": "10.0.3",
```

Comment:
This will stop it from working with <10 right?

Reply:
I have checked and modified the peerDependencies to set the minimal angular version to v7, because the v6 does not have the generic type on ModuleWithProviders interface.